src/adservice/src/main/java/hipstershop/AdService.java

```diff
@@ -49,7 +52,7 @@ public final class AdService {
   private static final AdService service = new AdService();
 
   private void start() throws IOException {
-    int port = Integer.parseInt(System.getenv().getOrDefault("PORT", "9555"));
+    int port = Integer.parseInt(System.getenv().getOrDefault("PORT", AD_SERVER_PORT+""));
```

Comment:
I personally find this code rather strange.
Suggest default service port use string 9555.
Thus avoiding int conversion. `AD_SERVER_PORT+""`

```suggestion
    int port = Integer.parseInt(System.getenv().getOrDefault("PORT", AD_SERVER_PORT));
```

Reply:
@wph95   
Do you really complie this code ? 
```
int port = Integer.parseInt(System.getenv().getOrDefault("PORT", AD_SERVER_PORT));
```
It will cause some errors 
```
Description	                                                        
The method getOrDefault(Object, String) in the type Map<String,String> is not applicable for the arguments (String, int)	
Resource  Path       Location                                   Type
AdService.java	/ad/src/main/java/hipstershop	line 55	   Java Problem
```
Maybe you are not familiar with Java Language ☺
Note :  The method ` System.getenv().getOrDefault ` requires  a String parameter ,  but The type `AD_SERVER_PORT` is int and need to transfer